eyeshade/controllers/publishers.js

```diff
@@ -59,6 +65,7 @@ v2.settlement = {
 
   validate: {
     payload: Joi.array().min(1).items(Joi.object().keys({
+      timestamp: braveJoi.date().iso().optional().description('the timestamp the settlement occurred'),
```

Comment:
what do you think about calling this `executed_at`?

Reply:
sure. yeah, happy to do any key

Reply:
@evq did you just want the input key to change or should it be stored on the document in that way too

Reply:
I think changing the field stored on the document would be good too :+1:

Reply:
would you be ok with `executedAt`? just to keep camel case consistent

Reply:
ah good point! sgtm :+1: